admin-api/authorization_server/internal_authorization.md

```diff
@@ -0,0 +1,95 @@
+---
+title: Internal authorization server
+menuTitle: Internal authorization server
+weight: 1
+---
+
+# Internal authorization server
+
+PrestaShop can be used as an authorization server, as such it can handle the permissions and authentication to access the resource server.
+
+When PrestaShop is used as the authorization server it doesn't need to perform an external call to validate the access token since it already has all the required data to do so in its database, but it still performs the same role.
+
+## Create API client
+
+Let's create our first client. You can add as many clients to the API as you want. In the real-world scenario, you probably want to create separate client for each service you want to integrate. You will probably have clients like "My ERP integration", "Some marketing automation tool client", etc.
+
+To add a new API client, navigate to Advanced Parameters -> Admin API page. Here you can see a list of all API clients, you can add a new one by clicking the "Add new API Client" button.
+
+![PrestaShop 9 API Client](../img/api_add_new_client1.jpeg)
+
+If you want to add a new client, you must provide information about it. In the form below, you must provide all the necessary information about the client. After that, you can click the "Generate client secret and save" button. This will generate a client secret for you and save the client in the database. **It's important to save the client secret because you won't be able to see it again.**
+
+![PrestaShop 9 Form for adding a new API client](../img/api_add_new_client2.jpeg)
+<p class="text-center text-muted small">Form for adding a new API client</p>
```

Comment:
```suggestion
### Form for adding a new API client
```

because this title is displayed as normal text

Reply:
This was copied from the blog article, but it doesn't make sense here I'll remove it completely